### PR TITLE
add a .travis.yml

### PR DESCRIPTION
**Pull request by ofrobots**

### CODE REVIEW

**Comment by ofrobots** [comment]
Build is broken w/ Node 0.12, but this should be merged regardless.

@GoogleCloudPlatform/node-team PTAL.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "0.12"
+  - "4"
+  - "5"
```
Comment (matthewloring) [inline .travis.yml]: We aren't testing against 5 in our other agents. Is there something special about error reporting that needs 5 integration?

**Comment by ofrobots** [comment]
Preemptively merging. TBR.

File: .travis.yml (hunk 1 above)
Comment (ofrobots) [inline .travis.yml]: Au contraire: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/.travis.yml

**Pull request merged**